style.css

```
@@ -402,6 +402,7 @@ li.event>p {
   margin: 0;
   padding: 0.5rem;
   display: flex;
+  flex-direction: column;
   align-items: center;
   column-gap: 0.5rem;
```

アイコンと名前の間のgap指定が縦並びの時にも効くよう、この部分だけ調整しちゃいましょう！

```suggestion
  gap: 0.5rem;
```